jiskefet-api-cpp.sh

```diff
@@ -0,0 +1,54 @@
+package: Jiskefet-Api-Cpp
+version: "%(tag_basename)s"
+tag: v0.2.2
+requires:
+  - boost
+  - "GCC-Toolchain:(?!osx)"
+  - cpprestsdk
+  - OpenSSL
+build_requires:
+  - CMake
+source: https://github.com/PascalBoeschoten/jiskefet-api-cpp
+incremental_recipe: |
+  make ${JOBS:+-j$JOBS} install
+  mkdir -p $INSTALLROOT/etc/modulefiles && rsync -a --delete etc/modulefiles/ $INSTALLROOT/etc/modulefiles
+---
+#!/bin/bash -ex
+
+case $ARCHITECTURE in
+    osx*) [[ ! $BOOST_ROOT ]] && BOOST_ROOT=$(brew --prefix boost);;
+esac
+
+cmake $SOURCEDIR                                    \
+      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT           \
+      ${BOOST_VERSION:+-DBOOST_ROOT=$BOOST_ROOT}    \
+      -DCPPREST_ROOT=${CPPRESTSDK_ROOT}          \
+      -DCPPREST_LIB=${CPPRESTSDK_ROOT}/lib64/libcpprest.so \
```

Comment:
This will fail on macOS as the path is `/lib/libcpprest.dylib` (not `/lib64/libcpprest.so`). Just provide `CPPREST_LIB` as a hint to `find_library(CPPREST_LIB cpprest)` in your CMake. CMake will locate the library.